### PR TITLE
Update Bitfinex symbol mapping + symbol properties database

### PR DESCRIPTION
**Pull request by StefanoRaggi**
#### Description
- Added Bitfinex symbol properties downloader (unit test)
- Updated Bitfinex symbol properties database
- Replaced usages of `BitfinexSymbolMapper` with `SymbolPropertiesDatabaseSymbolMapper`
- Removed `BitfinexSymbolMapper`

#### Related Issue
Includes #4708 and #4741 

#### Motivation and Context
- Support all available Bitfinex pairs
- Remove hardcoded list of Bitfinex tickers from LEAN code

#### Requires Documentation Change
No.

#### How Has This Been Tested?
- Brokerage unit tests

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [x] All new and existing tests passed.

### CODE REVIEW

File: Data/symbol-properties/symbol-properties-database.csv [hunk 1]
```diff
@@ -349,543 +349,397 @@ oanda,USDTRY,forex,USD/TRY,TRY,1,0.00001,1
...
-bitfinex,ZECBTC,crypto,Zcash-Bitcoin,BTC,1,0.00001,0.2
-bitfinex,ZECUSD,crypto,Zcash-US Dollar,USD,1,0.00001,0.2
-bitfinex,ZILBTC,crypto,Zilliqa-Bitcoin,BTC,1,0.00001,0.00000001
-bitfinex,ZILETH,crypto,Zilliqa-Ethereum,ETH,1,0.00001,0.00000001
-bitfinex,ZILUSD,crypto,Zilliqa-US Dollar,USD,1,0.00001,748.
-bitfinex,ZRXBTC,crypto,0x-Bitcoin,BTC,1,0.00001,20.
-bitfinex,ZRXDAI,crypto,0x-DAI (Multi-collateral),DAI,1,0.00001,0.00000001
-bitfinex,ZRXETH,crypto,0x-Ethereum,ETH,1,0.00001,20.
-bitfinex,ZRXUSD,crypto,0x-US Dollar,USD,1,0.00001,20.
+bitfinex,TESTAAATESTBBB,crypto,AAA-BBB,TESTBBB,1,0.00001,0.00000001,tAAABBB
```
Comment (Martin-Molinero) [inline Data/symbol-properties/symbol-properties-database.csv:446]: Not sure this should be here `TESTAAATESTBBB`?

**Review by Martin-Molinero — APPROVED** [review]
Nice 👍  minor find

File: Data/symbol-properties/symbol-properties-database.csv (hunk 1 above)
Comment (StefanoRaggi) [reply Data/symbol-properties/symbol-properties-database.csv:446]: Agree, will remove it

**Comment by Martin-Molinero** [comment]
Included by https://github.com/QuantConnect/Lean/pull/4888

**Pull request closed without merging**